postgresql/db-connection-status.yml

```diff
@@ -0,0 +1,38 @@
+type: db
+data:
+  query: WITH max_con AS
+      (
+        SELECT setting::float
+        FROM pg_settings
+        WHERE name = 'max_connections'
+      )
+    SELECT
+      MAX(setting) AS max_connections,
+      SUM(numbackends)/MAX(setting) AS pct_connections
+    FROM
+      pg_stat_database,
+      max_con;
+  dbUrl: jdbc:postgresql://${POSTGRESQL_DB_HOST_PORT}
+  dbDriverClass: org.postgresql.Driver
+  dbUser: ${POSTGRESQL_DB_USER}
+  dbPassword: ${POSTGRESQL_DB_PASSWORD}
+  dbAdditionalConnectionParams: ${POSTGRESQL_DB_ADDITIONAL_PARAMS}
+  #  dbVerticalModel="true" - because output of SHOW STATUS command gives each attribute in its own row
+  # (1st column being attribute name, 2nd column being attribute value)
+  dbVerticalModel: true
+
+observation:
+  - name: connection
+    metricNamespace: postgresql
+
+    metric:
+      - name: connections.max
```

Comment:
Here metric is named `connections.max`, while file `db-common-status.yml` has metrics like `database.connections.active.count` - note the `database` prefix. Are metrics from these two configs on different level? If metrics in `db-common-status.yml` are about specific database instance, are they missing some `database` tag?